src/style.css

```diff
@@ -0,0 +1,170 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form__field {
+  box-sizing: border-box;
+
+  position: absolute;
+  height: 70px;
+  left: 8px;
+  right: 8px;
+  top: 20px;
+  bottom: 0%;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.form__field:hover {
+  box-sizing: border-box;
+
+  position: absolute;
+  height: 70px;
+  left: 8px;
+  right: 8px;
+  top: 20px;
+  bottom: 0%;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.form__field:focus-within {
+  box-sizing: border-box;
+
+  position: absolute;
+  height: 70px;
+  left: 8px;
+  right: 8px;
+  top: 20px;
+  bottom: 0%;
+
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.small__form__field {
+  top: 110px;
+  height: 42px;
+}
+
+.small__form__field:hover {
+  top: 110px;
+  height: 42px;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.small__form__field:focus-within {
+  top: 110px;
+  height: 42px;
+
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.field {
+  position: absolute;
+
+  width: 158px;
+  height: 19px;
+  left: 61px;
+  top: 35.71%;
+  bottom: 37.14%;
+
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+
+  border: none;
+}
+
+::placeholder {
+  color: #3d4e61;
+}
+
+.field:focus {
+  position: absolute;
+
+  width: 158px;
+  height: 19px;
+  left: 62px;
+  top: 35.71%;
+  bottom: 37.14%;
+
+  font-family: Avenir, sans-serif;
+  font-weight: bold;
+  line-height: 19px;
```

Comment:
why do you add this styles on focus? 
you need to only add styles which are different